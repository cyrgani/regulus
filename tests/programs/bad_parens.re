# used to output is just "2", is a syntax error now
(
    print(2)
),
print(3)