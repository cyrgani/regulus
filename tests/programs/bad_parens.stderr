
SyntaxError: missing or invalid tokens for s_step